lib/src/patch.dart

```diff
@@ -11,10 +11,10 @@ class Patch {
   final FileSpan selection;
 
   /// Text to replace the selection with.
-  final String replacement;
+  String replacement;
```

Comment:
I really don't like making stuff mutable if it's at all possible to avoid it. It raises so much possibility for spooky action at a distance. I'd rather see a `Patch withReplacement(String replacement)` method that returns a new `Patch` with an different replacement string.